### PR TITLE
related widget wrapper select width depending on amount of links

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- it was not possible to solve this issue by css

|1 link | 2 links | 3 links |
|---|---|---|
|screen shot 2016-06-29 at 12 11 22|screen shot 2016-06-29 at 12 10 32|screen shot 2016-06-29 at 12 11 08|

### CODE REVIEW

File: djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+     * @function addsClassNames
+     */
+    function addsClassNames() {
+        var relatedWidgetWrappers = getRelatedWidgetWrappers();
+
+        relatedWidgetWrappers.forEach(function (wrapper) {
+            var links = wrapper.getElementsByClassName('related-widget-wrapper-link');
+
+            if (links.length === 1) {
+               wrapper.classList.add('widget-wrapper-links-1');
```
Comment (vxsx) [inline djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js]: don't use classList :(

File: djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            } else if (links.length === 2) {
+                wrapper.classList.add('widget-wrapper-links-2');
+            } else {
```
Comment (vxsx) [inline djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js]: can there be more?)

Comment (vxsx) [inline djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js]: i think it's safer to do something like this:

```js
if (links && links.length) {
    wrapper.className += ' widget-wrapper-links-' + links.length; 
}
```

Comment (lory87) [inline djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js]: no just add change and delete

File: djangocms_admin_style/templates/admin/base_site.html [hunk 3]
```diff
@@ -7,6 +7,7 @@
 <meta name="viewport" content="width=device-width,initial-scale=1" />
 <link rel="stylesheet" type="text/css" href="{% static 'djangocms_admin_style/css/djangocms-admin.css' %}" />
 <script src="{% static 'djangocms_admin_style/js/datetimefields.js' %}"></script>
+<script src="{% static 'djangocms_admin_style/js/related-widget-wrapper.js' %}"></script>
```
Comment (vxsx) [inline djangocms_admin_style/templates/admin/base_site.html]: don't really like duplication and multiple scripts. it's not in scope of this PR, but we should start bundling them together with webpack or something

File: djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js (hunk 1 above)
Comment (vxsx) [inline djangocms_admin_style/static/djangocms_admin_style/js/related-widget-wrapper.js]: also check linting, indentation is off

File: djangocms_admin_style/templates/admin/base_site.html (hunk 3 above)
Comment (lory87) [inline djangocms_admin_style/templates/admin/base_site.html]: yes I agree. Before there was no extra js file included. Do you create a task for this?

**Comment by lory87** [comment]
https://github.com/divio/djangocms-admin-style/issues/359

**Comment by vxsx** [comment]
👍

**Pull request merged**